src/Microsoft.AspNetCore.Cors/Infrastructure/CorsPolicy.cs

```diff
@@ -15,6 +16,14 @@ public class CorsPolicy
         private TimeSpan? _preflightMaxAge;
 
         /// <summary>
+        /// Default constructor for a CorsPolicy.
+        /// </summary>
+        public CorsPolicy()
+        {
+            IsOriginAllowed = DefaultIsOriginAllowed;
```

Comment:
Wondering - does the compiler allow initializing this assignment on the property itself? (C#6 property initializers)